src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -84,7 +84,7 @@ public ClaimDocument uploadDocument(String authorisation, PDF pdf) {
                 authorisation,
                 authTokenGenerator.generate(),
                 userDetails.getId(),
-                asList(userRoles),
+                asList(userRoles.split(",")),
```

Comment:
We don't have a test for list of roles?

Reply:
I think we could have chosen  `List<String> userRoles ` and bind coma separated properties. Please see example here:
https://github.com/hmcts/cmc-claim-store/blob/master/ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/ClaimMigrator.java#L35 .

Reply:
userRoles.split(",") -- the split() is not required .. please remove this

Reply:
We scratched our heads with @srineelesh to understand why removing the `split` works and it indeed does work! The update method interface forces us to supply a list so we are just going to comply to this and provide a list of seperate roles items - so keeping the split here is fine. @sabahirfan 's suggestion is a minor improvement we may want to implement when we roll out a better fox for this. I'll raise a story.

Reply:
👍

Reply:
@ckalista : either ways in the DM client that list is being converted to a comma separate string .. please have a look at https://github.com/hmcts/document-management-client/blob/f6709113c9c056f97ace73d3504951b314a73130/src/main/java/uk/gov/hmcts/reform/document/DocumentUploadClientApi.java#L108 . Please look into the history of this code to understand the issue more.